pages/index.js

```diff
@@ -36,38 +41,53 @@ class HomePage extends Component {
     }).isRequired,
     articles: ArticlesArray.isRequired,
     diary: DiaryShape.isRequired,
+    pagination: PropTypes.oneOfType([PropTypes.bool, PaginationShape]).isRequired,
```

Comment:
`bool`?

Reply:
It will be ``false`` when there are no more pages